### ISSUE TITLE
Bug in drq2ppt.py due to recent change

### ISSUE THREAD

**Issue by treerink** [EC-Earth/ece2cmor3#348]
@goord in 92e5907ab56f771776fe2ad26ec9ebddb7d33298 (which aims to fix issue #347) the line:
`````
freqgroups.pop(-1)
`````
leads to a crash for nearly all `drq2ppt.py` cases. Probably it goes only well if a `sub hourly` table is present.

**Comment by treerink** [comment]
In e253b48 I have commented the earlier mentioned line, this makes that drq2ppt.py is running again, but probably the #347 issue is back now.

I guess something like I check "if sub hourly present" needs to be added?

**Comment by treerink** [comment]
The error message belonging to the first post is:
`````
Traceback (most recent call last):
  File "./drq2ppt.py", line 297, in <module>
    main()
  File "./drq2ppt.py", line 290, in main
    write_ppt_files(ece2cmorlib.tasks)
  File "./drq2ppt.py", line 90, in write_ppt_files
    freqgroups.pop(-1)
KeyError: -1
`````

**Comment by goord** [comment]
Should be fixed in c3adf0eaf57084b66de719bf21a0eaf834d6718e

**Comment by treerink** [comment]
Yes now its ok, thanks
